frontend/test/metabase/scenarios/collections/personal-collections.cy.spec.js

```diff
@@ -179,3 +192,9 @@ function addNewCollection(name) {
   cy.findByLabelText("Name").type(name, { delay: 0 });
   cy.button("Create").click();
 }
+
+function openEllipsisMenuFor(item) {
```

Comment:
@nemanjaglumac since we're improving accessibility in our app, I'm not sure what's the best way to navigate table cells. I get the impression that how we write Cypress also determines how accessible our app is.

Am I overthinking this, or what we're doing here works just fine.

Reply:
Used the same thing we do in `collections.cy.spec.js`